security/backend/access_control/README.md

```diff
@@ -0,0 +1,46 @@
+# Access Control
+
+Access control has three significant parts: ensuring the identity of a user (Authentication), determining what rights they have in the application (Authorization), and maintaining some level of state as to the user’s identity (Session Management).
+
+## Authentication
+
+* Users must have individual user accounts with unique usernames.
+
+* Applications should use the single sign-on capabilities of its platform when possible instead of requiring users to make their own passwords.
+
+* The login page, login popup forms, the page before the login page, and all logged in pages must use TLS.
+
+* Following the Positive Security Model principle, all pages must require login unless specifically designated as public.
+
+* Applications should use the default authentication for their platform whenever possible.
+
+* Authentication controls must be enforced on a trusted system. For example, on the server rather than the client.
+
+* Applications must use only HTTP POST to transmit credentials.
+
+* Passwords must be at least 8 characters in length and should incorporate two alternative characters.
```

Comment:
I hesitate to include this. It is outdated guidance at this point.

For example, NIST has updated its [guidance](https://auth0.com/blog/dont-pass-on-the-new-nist-password-guidelines/). 

What about something like:

> Users should be encouraged and applications should support long, unique passwords.